CODE_OF_CONDUCT.md

```diff
@@ -60,8 +60,11 @@ representative at an online or offline event.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported to the community leaders responsible for enforcement at
-mptcp-owner@lists.01.org.
+reported to the community leaders responsible for enforcement at the
+e-mail address for the Linux kernel MPTCP subsystem owner listed in
+the kernel
+[https://github.com/multipath-tcp/mptcp_net-next/blob/export/MAINTAINERS][`MAINTAINERS`]
```

Comment:
Suggest linking to official Linux docs rather than mptcp_net-next:

https://www.kernel.org/doc/linux/MAINTAINERS